lib/src/messages/logical_replication_messages.dart

```diff
@@ -250,6 +252,10 @@ class RelationMessageColumn {
 }
 
 class RelationMessage implements LogicalReplicationMessage {
+  /// Stores the latest [RelationMessage]s for a given [relationId] to enable parsing of binary [TupleData]
+  /// from subsequent data modification messages.
+  static final Map<int, RelationMessage> _latest = {};
```

Comment:
First of all, let's not store such values in a static, non-scoped Map. If I would use two different connection to two different databases at the same time, this would become a conflicting bad data.

Question: is the `relationId` specific to the current client session, or is it a database-level ID stored  in some table and can be repeatedly accessed during a connection? If the later is the case, it should be cached at the Connection level.

Reply:
`relationId` is the `oid` from the `pg_class` table.